browser/fragments/ably-commonjs-epilogue.js

```diff
@@ -1,3 +1,8 @@
+var Types = {
+  ClientOptions: function(obj) { return (obj); }
```

Comment:
You're now duplicating code between this file and ably-epilogue?  Can we avoid that please?